Formula/guile.rb

```diff
@@ -43,18 +63,17 @@ def install
     # Avoid superenv shim
     inreplace "meta/guile-config.in", "@PKG_CONFIG@", Formula["pkg-config"].opt_bin/"pkg-config"
 
-    system "./autogen.sh" unless build.stable?
-
-    # Disable JIT on Apple Silicon, as it is not yet supported
-    # https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44505
-    extra_args = []
-    extra_args << "--enable-jit=no" if Hardware::CPU.arm?
+    if OS.mac?
+      # We need to regenerate ./configure because we just patch configure.ac.
+      system "autoreconf", "-vif"
+    else
+      system "./autogen.sh" unless build.stable?
+    end
```

Comment:
```suggestion
    system "./autogen.sh" unless build.stable?
    if OS.mac? && build.stable?
      # We need to regenerate ./configure because we just patch configure.ac.
      system "autoreconf", "-vif"
    end
```

Otherwise Mac head builds will be inconsistent.

Reply:
Makes sense.

Reply:
Done. Also added `--disable-nls` for the head builds since currently we add the gettext dependency but were not disabling it. This will make all builds consistent.